src/mobile/src/main/java/tupperdate/android/home/Buttons.kt

```diff
@@ -0,0 +1,51 @@
+package tupperdate.android.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.contentColor
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun BarButton(
+    size: Int,
+    onClick: () -> Unit,
+    backgroundColor: Color,
+    content: @Composable() () -> Unit
+) {
+    BarButton(
+        size = size, onClick = onClick, backgroundColor = backgroundColor,
+        contentColor = Color.White, content
+    )
+}
```

Comment:
You definitely want to refactor this, for a few reasons :

- You can use default parameters for arguments;
- The scope is a `RowScope`, which is more powerful than no scope at all; and
- Sizes MUST be specified in `Dp`, and SHOULD be passed as a `Modifier` parameter.

```suggestion
@Composable
fun BarButton(
    onClick: () -> Unit,
    modifier: Modifier = Modifier,
    foreground: Color = MaterialTheme.colors.primary,
    background: Color = contentColorFor(MaterialTheme.colors.primary),
    content: @Composable RowScope.() -> Unit
) {
   Button(
         modifier = modifier,
         shape = CircleShape,
         onClick = onClick,
         colors = ButtonConstants.defaultButtonColors(
             backgroundColor = background,
             contentColor = foreground
         ),
         // Actually, the prototypes have an elevation. This should be fixed.
         elevation = null
     ) {
         content()
     }
}
```